src/snovault/auditor.py

```diff
@@ -244,8 +244,9 @@ def item_view_audit(context, request):
     path = request.resource_path(context)
     properties = request.embed(path, '@@object')
     inherit = context.audit_inherit
-    if context.embedded and '*' in context.audit_inherit:
-        inherit = context.embedded
+    embedded_paths = list(context.embedded) + [p.path for p in context.embedded_with_frame]
```

Comment:
does `context.embedded_with_frame` raise an exception if the context doesn't have it set?